<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1884</width>
    <height>901</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Применение фильров</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/newPrefix/ZNNBQRDG_RA.jpg</normaloff>:/newPrefix/ZNNBQRDG_RA.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">@{background: rgb(255,255,255)};</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-color:rgb(255,255,255);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0,0,0">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>RGB</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Grey</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Параметры контура</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>391</width>
           <height>261</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0,0,0" columnstretch="0,0,0">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item row="3" column="2">
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Применить </string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Форма элемента</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Размер ядра</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>GreyNormalize</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Grey_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/newPrefix/ZNNBQRDG_RA.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_18">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Парамаметры фильтра</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="GreyNormalize_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/newPrefix/ZNNBQRDG_RA.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="RGB_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/newPrefix/ZNNBQRDG_RA.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>7</number>
      </property>
      <item row="1" column="3">
       <widget class="QLabel" name="Sobel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/newPrefix/ZNNBQRDG_RA.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Contour_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/newPrefix/ZNNBQRDG_RA.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="Sobel__">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Sobel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="Prevvit___">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Prevvit</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="Prevvit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/newPrefix/ZNNBQRDG_RA.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Canni">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>443</width>
          <height>358</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/newPrefix/ZNNBQRDG_RA.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="GreyLab2">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Canni</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Contour</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1884</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="download">
    <property name="title">
     <string>Загрузить изображение</string>
    </property>
    <addaction name="down_2"/>
   </widget>
   <widget class="QMenu" name="save">
    <property name="title">
     <string>Сохранить изображение</string>
    </property>
    <addaction name="RGB_3"/>
    <addaction name="Grey_3"/>
    <addaction name="GreyNormalize_3"/>
    <addaction name="Contour_3"/>
    <addaction name="Canni_3"/>
    <addaction name="Prevvit_3"/>
    <addaction name="Sobel_3"/>
   </widget>
   <addaction name="download"/>
   <addaction name="save"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="RGB_3">
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="down">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="down_2">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="Grey_3">
   <property name="text">
    <string>Grey</string>
   </property>
  </action>
  <action name="GreyNormalize_3">
   <property name="text">
    <string>GreyNormalize</string>
   </property>
  </action>
  <action name="Contour_3">
   <property name="text">
    <string>Contour</string>
   </property>
  </action>
  <action name="Canni_3">
   <property name="text">
    <string>Canni</string>
   </property>
  </action>
  <action name="Prevvit_3">
   <property name="text">
    <string>Prevvit</string>
   </property>
  </action>
  <action name="Sobel_3">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="greyHistrogram">
   <property name="text">
    <string>Гистограмма по серому цвету</string>
   </property>
  </action>
  <action name="greyContrastHist">
   <property name="text">
    <string>Гистограмма по серому цвету(контраст)</string>
   </property>
  </action>
  <action name="R_Hist">
   <property name="text">
    <string>Гистограмма(Канал R)</string>
   </property>
  </action>
  <action name="G_Hist">
   <property name="text">
    <string>Гистограмма(Канал G)</string>
   </property>
  </action>
  <action name="B_Hist">
   <property name="text">
    <string>Гистограмма(Канал B)</string>
   </property>
  </action>
  <action name="scaleM">
   <property name="text">
    <string>Масштабирование</string>
   </property>
  </action>
  <action name="L_Hist">
   <property name="text">
    <string>Гистограмма(Канал L)</string>
   </property>
  </action>
  <action name="u_Hist">
   <property name="text">
    <string>Гистограмма(Канал u)</string>
   </property>
  </action>
  <action name="v_Hist">
   <property name="text">
    <string>Гистограмма(Канал v)</string>
   </property>
  </action>
  <action name="Erode">
   <property name="text">
    <string>Эрозия </string>
   </property>
  </action>
  <action name="Dilate">
   <property name="text">
    <string>Дилатация</string>
   </property>
  </action>
  <action name="Opening">
   <property name="text">
    <string>Открытие</string>
   </property>
  </action>
  <action name="Closing">
   <property name="text">
    <string>Закрытие</string>
   </property>
  </action>
  <action name="Contour">
   <property name="text">
    <string>Параметры контура</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../.idea/Новый текстовый документ.qrc"/>
 </resources>
 <connections/>
</ui>
